components/admin_console/license_settings/renew_license_card/renew_license_card.tsx

```diff
@@ -0,0 +1,85 @@
+    let titleClass = 'RenewLicenseCard__text-title';
+    let idTitleText = 'admin.license.renewalCard.licenseExpiring';
+    let defaultTitleText = 'License expires in {days} days on {date, date, long}.';
+    let iconClass = 'icon-alert-circle-outline';
+    const today = moment(Date.now());
+    const endOfLicense = moment(new Date(parseInt(license?.ExpiresAt, 10)));
+    const daysToEndLicense = endOfLicense.diff(today, 'days');
+    if (isLicenseExpired) {
+        titleClass = 'RenewLicenseCard__text-title critical';
+        idTitleText = 'admin.license.renewalCard.licenseExpired';
+        defaultTitleText = 'License expired on {date, date, long}.';
+        iconClass = 'icon-alert-outline';
+    }
+    return (
+        <div className='RenewLicenseCard'>
+            <div className='RenewLicenseCard__text'>
+                <div className={titleClass}>
+                    <i className={iconClass}/>
+                    <FormattedMessage
+                        id={idTitleText}
+                        defaultMessage={defaultTitleText}
+                        values={{
+                            date: endOfLicense,
+                            days: daysToEndLicense,
+                        }}
+                    />
+                </div>
+                <div className='RenewLicenseCard__text-description bolder'>
+                    <FormattedMessage
+                        id='admin.license.renewalCard.description'
+                        defaultMessage='Renew your Enterprise license through the Customer Portal to avoid any disruption.'
+                    />
+                </div>
+                <div className='RenewLicenseCard__text-description'>
+                    <FormattedMessage
+                        id='admin.license.renewalCard.reviewNumbers'
+                        defaultMessage='Review your numbers below to ensure you renew for the right number of users.'
+                    />
+                </div>
+                <div className='RenewLicenseCard__licensedUsersNum'>
+                    <FormattedMarkdownMessage
+                        id='admin.license.renewalCard.licensedUsersNum'
+                        defaultMessage='**Licensed Users:** {licensedUsersNum}'
+                        values={{
+                            licensedUsersNum: license.Users,
+                        }}
+                    />
+                </div>
+                <div className='RenewLicenseCard__activeUsersNum'>
+                    <FormattedMarkdownMessage
+                        id='admin.license.renewalCard.usersNumbers'
+                        defaultMessage='**Active Users:** {activeUsersNum}'
+                        values={{
+                            activeUsersNum: totalUsers,
+                        }}
+                    />
+                </div>
+                <RenewalLink/>
```

Comment:
Hey @pablovelezvidal , thanks a lot for the good work. I just found some issues with this component here ```<RenewalLink/>```. 
> With a cloud license, we know that ```api/v4/license/renewal``` will not return a renewal link. This means in that component we don't set any link in the state. That leads to a somewhat misleading message on the modal. See image
<img width="1792" alt="Screenshot 2021-01-27 at 13 46 29" src="https://user-images.githubusercontent.com/28563179/105981206-e3f11f80-60a6-11eb-85c4-b2bea8fe2e15.png">

> Also, because the ```useEffect``` hook in that component runs once, changing the licence to like an E10 or E20 and then clicking the RenewalLink button will still show that misleading message of no internet connection

Reply:
thanks @AGMETEOR for your comments! I will check for more details behind the business logic of these use cases when it is a cloud license and when you change the type of license.

Reply:
Hi @AGMETEOR , thanks again for the review! so, no Cloud customer is supposed to see the Edition and License screen, meaning that the scenario about receiving a bad request for the renewal link is only happening to us in development for a special configuration that we use to see this page. Also, we want to keep it that way for the hypothetical scenario where our Cloud License is expiring and this way we can easily see that instead of it to fail silently. The other scenario mentioned is not a problem, because every time that you remove/add a license, the entire component and child tree is re-rendered, see attached gif.

![licenseCard](https://user-images.githubusercontent.com/10082627/106126900-4527fa00-615e-11eb-9ae7-c822bf644787.gif)

Reply:
Alright. Thanks a lot @pablovelezvidal